var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -186,6 +197,12 @@ def make_defs(self):
         if '+ilp64' in self.spec:
             make_defs += ['INTERFACE64=1']
 
+        if 'x86' in spack.architecture.sys_type():
```

Comment:
Ideally this would examine the spec rather than the architecture of the system (since some systems support multiple targets or do cross compiling). But this pattern is already followed in a few other places (other packages, and the openblas definition). Looking at the spec would look like:

`if 'aarch64' in spec.architecture.target.lower()`